yarn.lock

```diff
@@ -4028,8 +4028,8 @@ joi@^13.0.0:
 
 "js-owncloud-client@github:owncloud/js-owncloud-client#master":
   version "0.1.3"
-  uid "5bcbfbc926259673e1208b8cab99fa8f88112dbd"
-  resolved "https://codeload.github.com/owncloud/js-owncloud-client/tar.gz/5bcbfbc926259673e1208b8cab99fa8f88112dbd"
+  uid "9aa61638239b95bffb2f31fabfd3e6562403f55e"
+  resolved "https://github.com/owncloud/js-owncloud-client#9aa61638239b95bffb2f31fabfd3e6562403f55e"
```

Comment:
@dependabot-bot this is wrong - should look like this:

```
"js-owncloud-client@github:owncloud/js-owncloud-client#master":
  version "0.1.3"
  resolved "https://codeload.github.com/owncloud/js-owncloud-client/tar.gz/9aa61638239b95bffb2f31fabfd3e6562403f55e"
  dependencies:
    promise "^8.0.0"
    request "^2.81.0"
    utf8 "^3.0.0"
    xml-js "^1.3.2"

Reply:
@DeepDiver1975 yes on it! Looks like we're not dealing with `@github:` installs correctly